tools/flaky-test-reporter/jsonreport/jsonreport.go

```diff
@@ -36,7 +37,11 @@ const (
 
 // Report contains concise information about current flaky tests
 type Report struct {
-	Repo  string   `json:"-"`
+	Reports []RepoReport
```

Comment:
whats the point of this struct? Doesnt look like we are using it here anywhere?

Reply:
GetFlakyTestReport() should return one, looks like I wrote it to populate the report you call it on. Will fix.

Reply:
why cant GetFlakyTestReport() return []Report instead of this struct?

Reply:
very good point, I have no plans to use the Report struct in other places. I can fix this.